.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: rust
+rust:
```

Comment:
@reem I removed 1.1.0 since travis downloads the latest stable rust release by itself, but nightly has added and requested changes have been made